### PR TITLE
Portfolio: mobile menu

### PR DESCRIPTION
**Pull request by edi-sipka**
In this pull request me and @gnuion added:
- Closing and opening mobile menu in javascript
- Closing mobile menu when clicking on any li element
- Add navigation through section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,29 +10,53 @@
...
           <h2 class="my-logo">My Logo</h2>
         </div>
       </a>
-
-      <nav class="mobile-menu">
-        <div class="navigation-right">
-          <img src="Images/Enabled.png" alt="mobile menu" />
+      <nav>
+        <div class="nav-right">
+          <img src="images/Enabled.png" alt="normal button" />
```
Comment (omarsalem7) [inline index.html:50]: - [ ] Kindly fix the image path because the image directory starts with the Capital `I`.

File: style.css [hunk 2]
```diff
@@ -621,4 +606,70 @@ header {
...
+  position: absolute;
+  background: #6070ffee;
+  mix-blend-mode: multiply;
+  backdrop-filter: blur(8px);
+  top: 0;
+  left: 0;
+  height: 100vh;
+  width: 100vw;
+  display: none;
+}
```
Comment (omarsalem7) [inline style.css:626]: - [ ] Kindly add a blur effect for the content behind the menu like in the Figma design.

| Your design  | Figma design|
| ------------- | ------------- |
| image | image |

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @edi-sipka, @gnuion 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Descriptive pull request ✔️
- No linter errors ✔️
- Open/close functionality works correctly ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codecaiine — APPROVED** [review]
Hi @TEAM, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Descriptive Pull request :heavy_check_mark: 
- Good use of Gitflow :heavy_check_mark: 
- Commits are meaningful :heavy_check_mark: 
- Clean code :heavy_check_mark:  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**